test/integration/marked-spec.js

```diff
@@ -1,5 +1,133 @@
 var marked = require('../../marked.min.js');
 
-it('should run the test', function () {
-  expect(marked('Hello World!')).toBe('<p>Hello World!</p>\n');
+describe('integration suite', function () {
+  it('should put plaintext into a paragraph', function () {
+    expect(marked('Hello World!')).toBe('<p>Hello World!</p>\n');
+  });
+
+  describe('special characters', function () {
+    it('should handle ampersands', function () {
+      var samples = [{'md': 'AT&T has an ampersand in their name', 'html': '<p>AT&amp;T has an ampersand in their name</p>\n'},
+                     {'md': 'AT&amp;T is another way to write it', 'html': '<p>AT&amp;T is another way to write it</p>\n'},
+                     {'md': 'This & that', 'html': '<p>This &amp; that</p>\n'},
+                     {'md': 'Inline [link](/script?foo=1&bar=2) with ampersands', 'html': '<p>Inline <a href="/script?foo=1&amp;bar=2">link</a> with ampersands</p>\n'},
+                     {'md': '[link with ampersands](http://example.com/?foo=1&bar=)', 'html': '<p><a href="http://example.com/?foo=1&amp;bar=">link with ampersands</a></p>\n'},
+                     {'md': '[link title with ampersands](http://att.com "AT&T")', 'html': '<p><a href="http://att.com" title="AT&amp;T">link title with ampersands</a></p>\n'},
+                    ];
```

Comment:
I think it would be easier to just pull these from their current files. I hate hard coding strings in javascript. Too much possibility for errors.

We could construct each test from the files and add the `describe` and `it` descriptions to the front-matter in the .md file

Reply:
or maybe reorganize these tests and pull the `describe` description from the folder name